android/app/src/main/res/layout/search_item_speaker.xml

```diff
@@ -0,0 +1,72 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
@mananwason I have tried to reuse the layouts but the issue was that the layout was improper for the search page. So had to restructure the same layout for the search page.
As i mentioned earlier in this thread that the layout for the Speakers Fragment is more suited for a grid layout and not for a linear layout manager.